src/main/kotlin/org/kamiblue/client/util/text/MessageSendHelper.kt

```diff
@@ -16,6 +17,13 @@ import java.util.regex.Pattern
 object MessageSendHelper {
     private val mc = Wrapper.minecraft
 
+    fun sendChatMessage(message: ITextComponent) {
+        val component = TextComponentString("")
+        component.appendSibling(ChatMessage(coloredName('9')))
+        component.appendSibling(message)
+        sendRawChatMessage(component)
+    }
```

Comment:
What is `9` in this case, and why is it hardcoded?

Reply:
This is to replicate just the normal chat message, it means that it isn't an error. Look at the rest of messageSendHelper for other examples.